### PR TITLE
Big decimal truncation

### PR DESCRIPTION
**Pull request by paulwarren-wk**
This PR avoids decimals with a large number of significant figures from being truncated when displayed in the fact inspector.

The last three [new tests](https://github.com/Workiva/ixbrl-viewer/compare/master...paulwarren-wk:ixbrl-viewer:big-decimal-truncation?expand=1#diff-4abe2da922e1dfef10090104939b6a0292fad5a67f35a09a27fa3528f7785e71R100) in this PR all fail under the old code.

This adds a dependency on `decimal.js`. I have another branch with draft support for Calculations 1.1 which also relies on `decimal.js`, so the new dependency will not be exclusively for this.

This fixes XT-1279

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by austinmatherne-wk — APPROVED** [review]

**Review by aaroncameron-wk — APPROVED** [review]

File: iXBRLViewerPlugin/viewer/src/js/util.js [hunk 1]
```diff
@@ -39,10 +40,12 @@ export function momentToHuman(d, adjust) {
...
  */
-export function formatNumber(v, d) {
-    var n = Number(v);
-    var s = d === undefined ? n.toString() : n.toFixed(Math.max(0, d));
-    return s.replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ",");
+export function formatNumber(value, decimals) {
+    const n = Decimal(value);
+    const s = decimals === undefined ? n.toFixed() : n.toFixed(Math.max(0, decimals));
+    const parts = s.split('.');
+    parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```
Comment (brettkail-wk) [inline iXBRLViewerPlugin/viewer/src/js/util.js:47]: Long-term, I wonder if it's worth trying to use something like `Intl.NumberFormat`.

**Review by brettkail-wk — APPROVED** [review]

**Comment by austinmatherne-wk** [comment]
# Verified

* large decimal values are no longer truncated in the fact inspector.

QA +10
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**